users/actions/activate_user.py

```diff
@@ -4,21 +4,20 @@
 from users.models import User
 
 
-class ActivateUser:
+class ActivateUser(object):
```

Comment:
What's the point? Taken from readme:
`Create virtualenv for python 3.5.`

Anyway, it is not even python 3 idea. This is Python 2.1 (!) convention, which is deprecated now:
https://wiki.python.org/moin/NewClassVsClassicClass